lib/reindex/reporting.py

```diff
@@ -0,0 +1,89 @@
+import requests
+import redis
+import json
+import ast
+import sys
+import time
+import urllib
+import re
+import sys
+from threading import Thread
+from concurrent.futures import ThreadPoolExecutor
+
+if len(sys.argv) == 2:
+    ip = sys.argv[1]
+    print("Sentinel IP used %s" % ip)
+else:
+    ip = "127.0.0.1"
+
+
+def safe_print(content):
+    print("{0}".format(content))
+
+
+class askRedis():
```

Comment:
TIL: use of parens for class definitions in python are a thing o.o

Reply:
I think it is unnecessary but —the parens may be used for class inheritance. https://docs.python.org/3.4/tutorial/classes.html#inheritance.

Reply:
yea, it was interesting to see, I guess introduced in python3